prac_05/emails.py

```diff
@@ -0,0 +1,19 @@
+"""
+emails
+Estimate: 60 minutes
+Actual:    minutes
+"""
+
+store_email_name = {}
+email = str(input("Email: "))
+while email != "":
+    email_parts = [str(parts) for parts in email.split("@")]
+    whole_name = " ".join(email_parts[0].split(".")).title()
+    print(whole_name)
```

Comment:
This print statement may not be required. With it, the output does not match sample output in prac